src/styles/auto-prefix.js

```diff
@@ -28,7 +28,7 @@ export default {
         userAgent: userAgent,
       });
 
-      return prefixer.prefix;
+      return prefixer;
```

Comment:
From what I understood from the issues you mentioned. It has a reference issue, wouldn't 

```js
return (style) => prefixer.prefix(style);
```
fix it?